pkg/log/options.go

```diff
@@ -82,22 +94,6 @@ type Options struct {
 	outputLevel     string
```

Comment:
Update these to `OutputLevel? At a minimum you'd avoid the reinterpretation to string in a few methods below.

Reply:
I originally tried that, but my lack of golang chops was hindering me.  Trouble figuring out how to pass a reference to these fields `cmd.PersistentFlags().StringVar`.  Any ideas?

Reply:
Ah, didn't realize it was used there - you could probably do it with some casting, but it's fine as-is.

Reply:
Yeah, just not sure the right way to cast here to make it work. Either way, the type is an implementation detail and doesn't affect the interface. We can always change it later.